py3status/modules/net_rate.py

```diff
@@ -5,41 +5,51 @@
 Configuration parameters:
     all_interfaces: ignore self.interfaces, but not self.interfaces_blacklist
         (default True)
-    cache_timeout: how often we refresh this module in seconds (default 2)
-    devfile: location of dev file under /proc (default '/proc/net/dev')
-    format: format of the module output (default '{interface}: {total}')
-    format_no_connection: when there is no data transmitted from the
-        start of the plugin (default '')
-    hide_if_zero: hide indicator if rate == 0 (default False)
-    interfaces: comma separated list of interfaces to track (default '')
+    cache_timeout: how often we refresh this module in seconds
+        (default 2)
+    devfile: location of dev file under /proc
+        (default '/proc/net/dev')
+    format: format of the module output
+        (default '{interface}: {total}')
+    format_no_connection: when there is no data transmitted from the start of the plugin
+        (default '')
+    hide_if_zero: hide indicator if rate == 0
+        (default False)
+    interfaces: comma separated list of interfaces to track
+        (default '')
```

Comment:
I can do that. By default, if it isn't a list, the module will try to split at `,` for backward compatibility